### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/PHSEND/CGINS-PHSEND-P0068__20231102.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0068,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0068_Validation_Files_2023-11-02.zip > PHSEN-D_SN_P0068_CalCert.pdf
+P0068,CC_ea578,94.1,
+P0068,CC_eb434,2284.0,
+P0068,CC_eb578,38676.0,
+P0068,CC_ind_off,0,
+P0068,CC_ind_slp,1,
+P0068,CC_psal,35,Constant
+P0068,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0068__20231102.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0099__20231120.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0099,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0099_Validation_Files_2023-11-20.zip > SAMI_P0099_201123t_ValCert.pdf
+P0099,CC_ea578,94.1,
+P0099,CC_eb434,2284.0,
+P0099,CC_eb578,38676.0,
+P0099,CC_ind_off,0,
+P0099,CC_ind_slp,1,
+P0099,CC_psal,35,Constant
+P0099,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0099__20231120.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0100__20231120.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0100,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0100_Validation_Files_2023-11-20.zip > SAMI_P0100_201123t_ValCert.pdf
+P0100,CC_ea578,94.1,
+P0100,CC_eb434,2284.0,
+P0100,CC_eb578,38676.0,
+P0100,CC_ind_off,0,
+P0100,CC_ind_slp,1,
+P0100,CC_psal,35,Constant
+P0100,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0100__20231120.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0155__20231102.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0155,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0155_Validation_Files_2023-11-02.zip > PHSEN-D_SN_P0155_CalCert.pdf
+P0155,CC_ea578,94.1,
+P0155,CC_eb434,2284.0,
+P0155,CC_eb578,38676.0,
+P0155,CC_ind_off,0,
+P0155,CC_ind_slp,1,
+P0155,CC_psal,35,Constant
+P0155,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0155__20231102.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0156__20231107.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0156,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0156_Validation_Files_2023-11-07.zip > SAMI_P0156_071123t_ValCert.pdf
+P0156,CC_ea578,94.1,
+P0156,CC_eb434,2284.0,
+P0156,CC_eb578,38676.0,
+P0156,CC_ind_off,0,
+P0156,CC_ind_slp,1,
+P0156,CC_psal,35,Constant
+P0156,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0156__20231107.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0158__20231102.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0158,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0158_Validation_Files_2023-11-02.zip > PHSEN-D_SN_P0158_CalCert.pdf
+P0158,CC_ea578,94.1,
+P0158,CC_eb434,2284.0,
+P0158,CC_eb578,38676.0,
+P0158,CC_ind_off,0,
+P0158,CC_ind_slp,1,
+P0158,CC_psal,35,Constant
+P0158,CC_sami_bits,12,
```
Comment (cooleyky) [inline calibration/PHSEND/CGINS-PHSEND-P0158__20231102.csv:9]: looks good

**Review by cooleyky — COMMENTED** [review]
Completed review of PCO2WB and PHSEND cal csvs. Potential change needed on CGINS-PCO2WB-C0098__20231121.csv where end of cal range (line 11) doesn't match number in calibration certificate.

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20231103.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
....36, 273.18, 273.99, 274.8, 275.61, 276.42, 277.24, 278.05, 278.86, 279.67, 280.48, 281.3, 282.11, 282.92, 283.73, 284.54, 285.36, 286.17, 286.98, 287.79, 288.6, 289.42, 290.23, 291.04, 291.85, 292.66, 293.48, 294.29, 295.1, 295.91, 296.72, 297.53, 298.34, 299.16, 299.97, 300.78, 301.59, 302.4, 303.21, 304.02, 304.83, 305.65, 306.46, 307.27, 308.08, 308.89, 309.7, 310.51, 311.32, 312.13, 312.94, 313.75, 314.56, 315.37, 316.18, 316.99, 317.8, 318.61, 319.42, 320.23, 321.04, 321.85, 322.66, 323.47, 324.27, 325.08, 325.89, 326.7, 327.51, 328.32, 329.13, 329.93, 330.74, 331.55, 332.36, 333.16, 333.97, 334.78, 335.59, 336.39, 337.2, 338.01, 338.81, 339.62, 340.43, 341.23, 342.04, 342.84, 343.65, 344.46, 345.26, 346.07, 346.87, 347.68, 348.48, 349.29, 350.09, 350.9, 351.7, 352.5, 353.31, 354.11, 354.91, 355.72, 356.52, 357.32, 358.13, 358.93, 359.73, 360.53, 361.33, 362.14, 362.94, 363.74, 364.54, 365.34, 366.14, 366.94, 367.74, 368.54, 369.34, 370.14, 370.94, 371.74, 372.54, 373.34, 374.14, 374.94, 375.73, 376.53, 377.33, 378.13, 378.92, 379.72, 380.52, 381.31, 382.11, 382.91, 383.7, 384.5, 385.29, 386.09, 386.88, 387.68, 388.47, 389.26, 390.06, 390.85, 391.65, 392.44, 393.23, 394.02]",
```
Comment (cooleyky) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20231103.csv:8]: spot checked, looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20231027.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
...3.29, 274.1, 274.92, 275.73, 276.54, 277.36, 278.17, 278.98, 279.8, 280.61, 281.42, 282.24, 283.05, 283.87, 284.68, 285.49, 286.31, 287.12, 287.93, 288.75, 289.56, 290.37, 291.19, 292, 292.81, 293.63, 294.44, 295.25, 296.07, 296.88, 297.69, 298.51, 299.32, 300.13, 300.95, 301.76, 302.57, 303.38, 304.2, 305.01, 305.82, 306.64, 307.45, 308.26, 309.07, 309.89, 310.7, 311.51, 312.32, 313.13, 313.95, 314.76, 315.57, 316.38, 317.19, 318, 318.82, 319.63, 320.44, 321.25, 322.06, 322.87, 323.68, 324.49, 325.3, 326.11, 326.92, 327.73, 328.55, 329.36, 330.17, 330.98, 331.78, 332.59, 333.4, 334.21, 335.02, 335.83, 336.64, 337.45, 338.26, 339.07, 339.88, 340.68, 341.49, 342.3, 343.11, 343.92, 344.72, 345.53, 346.34, 347.14, 347.95, 348.76, 349.57, 350.37, 351.18, 351.98, 352.79, 353.6, 354.4, 355.21, 356.01, 356.82, 357.62, 358.43, 359.23, 360.04, 360.84, 361.64, 362.45, 363.25, 364.05, 364.86, 365.66, 366.46, 367.27, 368.07, 368.87, 369.67, 370.47, 371.28, 372.08, 372.88, 373.68, 374.48, 375.28, 376.08, 376.88, 377.68, 378.48, 379.28, 380.08, 380.88, 381.68, 382.47, 383.27, 384.07, 384.87, 385.67, 386.46, 387.26, 388.06, 388.85, 389.65, 390.45, 391.24, 392.04, 392.83, 393.63, 394.42, 395.22]",
```
Comment (cooleyky) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20231027.csv:8]: spot checked, looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01102__20231026.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
....86, 274.67, 275.48, 276.3, 277.11, 277.92, 278.73, 279.55, 280.36, 281.17, 281.98, 282.8, 283.61, 284.42, 285.23, 286.05, 286.86, 287.67, 288.48, 289.3, 290.11, 290.92, 291.74, 292.55, 293.36, 294.17, 294.98, 295.8, 296.61, 297.42, 298.23, 299.05, 299.86, 300.67, 301.48, 302.3, 303.11, 303.92, 304.73, 305.54, 306.35, 307.17, 307.98, 308.79, 309.6, 310.41, 311.22, 312.04, 312.85, 313.66, 314.47, 315.28, 316.09, 316.9, 317.71, 318.53, 319.34, 320.15, 320.96, 321.77, 322.58, 323.39, 324.2, 325.01, 325.82, 326.63, 327.44, 328.25, 329.06, 329.87, 330.68, 331.49, 332.29, 333.1, 333.91, 334.72, 335.53, 336.34, 337.15, 337.96, 338.76, 339.57, 340.38, 341.19, 341.99, 342.8, 343.61, 344.42, 345.22, 346.03, 346.84, 347.64, 348.45, 349.26, 350.06, 350.87, 351.68, 352.48, 353.29, 354.09, 354.9, 355.7, 356.51, 357.31, 358.12, 358.92, 359.72, 360.53, 361.33, 362.14, 362.94, 363.74, 364.55, 365.35, 366.15, 366.95, 367.76, 368.56, 369.36, 370.16, 370.96, 371.76, 372.57, 373.37, 374.17, 374.97, 375.77, 376.57, 377.37, 378.17, 378.97, 379.77, 380.56, 381.36, 382.16, 382.96, 383.76, 384.56, 385.35, 386.15, 386.95, 387.74, 388.54, 389.34, 390.13, 390.93, 391.72, 392.52, 393.31, 394.11, 394.9, 395.7]",
```
Comment (cooleyky) [inline calibration/NUTNRB/CGINS-NUTNRB-01102__20231026.csv:8]: spot checked, looks good

**Review by cooleyky — APPROVED** [review]
everything looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20231106.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20231106.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20231103.csv (hunk 7 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20231103.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20231027.csv (hunk 8 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20231027.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01102__20231026.csv (hunk 9 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01102__20231026.csv:8]: spot checked - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20231109.csv [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+ACS-206,CC_tbins,"[1.406081, 2.455000, 3.439111, 4.461429, 5.481818, 6.492857, 7.505769, 8.510870, 9.485000, 10.496667, 11.483043, 12.480800, 13.494800, 14.487083, 15.496800, 16.492800, 17.484615, 18.508889, 19.483333, 20.486000, 21.492273, 22.510833, 23.498000, 24.495600, 25.497407, 26.493846, 27.499600, 28.494167, 29.490741, 30.496538, 31.510385, 32.501379, 33.500000, 34.509615]",
+ACS-206,CC_taarray,SheetRef:CC_taarray,
+ACS-206,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20231109.csv:9]: spot checked - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20231109__CC_taarray.ext [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+-0.010510,-0.007834,-0.006919,-0.006411,-0.005940,-0.005672,-0.005485,-0.005142,-0.004606,-0.004110,-0.004076,-0.003755,-0.003375,-0.003037,-0.002687,-0.002285,-0.001915,-0.001449,-0.001289,-0.001376,-0.001021,-0.000643,-0.000440,-0.000087,0.000000,0.000094,0.000499,0.000375,0.000817,0.000697,0.000965,0.001007,0.001089,0.000841
+-0.010830,-0.008062,-0.007227,-0.006707,-0.006050,-0.005819,-0.005690,-0.005258,-0.004665,-0.004245,-0.004175,-0.003793,-0.003531,-0.003134,-0.002727,-0.002257,-0.001897,-0.001512,-0.001328,-0.001239,-0.000973,-0.000550,-0.000394,-0.000166,0.000000,0.000141,0.000568,0.000317,0.000947,0.000724,0.001106,0.001210,0.001060,0.000872
+-0.010656,-0.007990,-0.007176,-0.006637,-0.005897,-0.005798,-0.005723,-0.005136,-0.004678,-0.004212,-0.004226,-0.003921,-0.003667,-0.003175,-0.002790,-0.002456,-0.002072,-0.001504,-0.001500,-0.001543,-0.001133,-0.000747,-0.000579,-0.000271,0.000000,0.000080,0.000345,0.000130,0.000762,0.000533,0.000974,0.001017,0.000890,0.000637
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20231109__CC_taarray.ext:87]: spot checked - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00206__20231109__CC_tcarray.ext [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+-0.002102,-0.000080,0.000831,0.000949,0.001015,0.001167,0.000487,0.000260,0.000454,0.000361,0.000465,0.000652,0.000542,0.000761,0.000513,0.000561,0.000530,0.000420,0.000575,0.000410,0.000317,0.000095,0.000141,0.000079,0.000000,-0.000258,-0.000550,-0.000540,-0.001104,-0.001307,-0.001403,-0.001797,-0.002343,-0.002510
+-0.002725,-0.000028,0.000171,0.000195,0.000352,0.000721,0.000141,0.000419,0.000551,0.000889,0.000591,0.000824,0.000761,0.001010,0.000737,0.000815,0.000664,0.000634,0.000337,0.000585,0.000415,0.000266,-0.000040,0.000029,0.000000,-0.000242,-0.000517,-0.000582,-0.000888,-0.001176,-0.001373,-0.001644,-0.002104,-0.002346
+-0.001360,0.000334,0.001003,0.001476,0.001752,0.001905,0.001303,0.001230,0.001205,0.001048,0.000950,0.001120,0.000871,0.001045,0.000914,0.000555,0.000693,0.000497,0.000530,0.000653,0.000401,0.000320,0.000207,0.000346,0.000000,-0.000212,-0.000381,-0.000630,-0.000818,-0.000962,-0.001139,-0.001606,-0.001909,-0.002302
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00206__20231109__CC_tcarray.ext:87]: spot checked - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00218__20231110.csv [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+ACS-218,CC_tbins,"[0.688506, 1.380150, 2.448200, 3.469032, 4.492083, 5.497500, 6.544091, 7.503600, 8.502174, 9.499583, 10.507826, 11.495417, 12.498800, 13.496667, 14.497200, 15.496800, 16.500000, 17.502400, 18.502692, 19.489583, 20.496800, 21.495600, 22.490000, 23.491250, 24.479600, 25.487692, 26.497200, 27.503846, 28.490400, 29.502963, 30.495185, 31.491071, 32.491000, 33.499032, 34.523889]",
+ACS-218,CC_taarray,SheetRef:CC_taarray,
+ACS-218,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00218__20231110.csv:9]: spot checked - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00218__20231110__CC_taarray.ext [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+-0.006055,-0.006790,-0.006413,-0.005858,-0.005686,-0.005774,-0.005025,-0.005050,-0.005201,-0.003935,-0.003533,-0.003100,-0.002815,-0.002655,-0.002539,-0.002272,-0.002390,-0.001680,-0.002189,-0.001762,-0.001343,-0.001135,-0.000920,-0.000765,-0.000956,0.000000,-0.000364,-0.000120,0.000091,0.000172,0.000771,0.000619,0.000604,0.000831,0.000355
+-0.005478,-0.006419,-0.006089,-0.005582,-0.005321,-0.005507,-0.004797,-0.004873,-0.004926,-0.003828,-0.003353,-0.002858,-0.002565,-0.002380,-0.002370,-0.002036,-0.002300,-0.001457,-0.002148,-0.001567,-0.001228,-0.000921,-0.000865,-0.000800,-0.000999,0.000000,-0.000448,-0.000177,0.000100,0.000294,0.000884,0.000707,0.000988,0.000815,0.000503
+-0.006215,-0.007161,-0.006823,-0.006253,-0.006002,-0.006197,-0.005430,-0.005479,-0.005525,-0.004227,-0.003735,-0.003173,-0.002964,-0.002619,-0.002723,-0.002435,-0.002691,-0.001755,-0.002367,-0.001892,-0.001542,-0.001302,-0.001197,-0.000893,-0.001197,0.000000,-0.000495,-0.000291,-0.000073,-0.000056,0.000712,0.000428,0.000562,0.000737,0.000293
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00218__20231110__CC_taarray.ext:89]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00218__20231110__CC_tcarray.ext [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+-0.007017,-0.001790,0.000596,0.001204,0.001364,0.000876,-0.000016,0.000734,0.000402,0.000738,0.000540,0.000796,0.000870,0.000421,0.000369,0.000732,0.000661,0.000689,0.000931,0.000563,0.000502,0.000474,0.000291,0.000071,0.000076,0.000000,-0.000449,-0.000764,-0.001199,-0.001466,-0.001754,-0.002073,-0.002946,-0.003425,-0.004592
+-0.008151,-0.003068,-0.000624,-0.000085,0.000072,-0.000192,-0.000916,-0.000179,-0.000532,-0.000206,-0.000394,-0.000121,0.000211,-0.000083,-0.000188,0.000192,0.000013,0.000196,0.000283,0.000070,0.000443,0.000284,0.000244,-0.000109,-0.000040,0.000000,-0.000466,-0.000846,-0.001193,-0.001481,-0.001538,-0.002026,-0.002568,-0.002930,-0.004157
+-0.009040,-0.004115,-0.001905,-0.001146,-0.001079,-0.001350,-0.002060,-0.001289,-0.001380,-0.000984,-0.001446,-0.000596,-0.000287,-0.000774,-0.000657,-0.000207,-0.000211,-0.000054,-0.000031,-0.000275,-0.000046,0.000154,0.000061,-0.000299,-0.000165,0.000000,-0.000348,-0.000687,-0.001041,-0.001373,-0.001469,-0.001601,-0.002462,-0.002897,-0.004064
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00218__20231110__CC_tcarray.ext:89]: spot checked, looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0096__20231121.csv [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+C0096,CC_cala,0.0143,source_file PCO2W-B_SAMI-CO2_SN_C0096_Calibration_Files_2023-11-21.zip > SAMI_211123_CalCert_C0096.pdf
+C0096,CC_calb,0.7784,
+C0096,CC_calc,-1.8153,
+C0096,CC_calt,11.0202,
+C0096,CC_ea434,19706,Constant
+C0096,CC_ea620,34,Constant
+C0096,CC_eb434,3073,Constant
+C0096,CC_eb620,44327,Constant
+C0096,CC_sami_bits,12,
+C0096,CC_cal_range,"[204,988]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0096__20231121.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0097__20231121.csv [hunk 18]
```diff
@@ -0,0 +1,11 @@
...
+C0097,CC_cala,0.0669, source_file PCO2W-B_SAMI-CO2_SN_C0097_Calibration_Files_2023-11-21.zip > SAMI_211123_CalCert_C0097.pdf
+C0097,CC_calb,0.5067,
+C0097,CC_calc,-1.4799,
+C0097,CC_calt,10.7674,
+C0097,CC_ea434,19706,constant
+C0097,CC_ea620,34,constant
+C0097,CC_eb434,3073,constant
+C0097,CC_eb620,44327,constant
+C0097,CC_sami_bits,12,
+C0097,CC_cal_range,"[204,988]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0097__20231121.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0098__20231121.csv [hunk 19]
```diff
@@ -0,0 +1,11 @@
...
+C0098,CC_cala,0.0608,source_file PCO2W-B_SAMI-CO2_SN_C0098_Calibration_Files_2023-11-21.zip > C0098_CalCert.pdf
+C0098,CC_calb,0.5292,
+C0098,CC_calc,-1.4578,
+C0098,CC_calt,10.7398,
+C0098,CC_ea434,19706,constant
+C0098,CC_ea620,34,constant
+C0098,CC_eb434,3073,constant
+C0098,CC_eb620,44327,constant
+C0098,CC_sami_bits,12,
+C0098,CC_cal_range,"[204,988]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0098__20231121.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0100__20231121.csv [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+C0100,CC_cala,0.0508,source_file PCO2W-B_SAMI-CO2_SN_C0100_Calibration_Files_2023-11-21.zip > C0100_CalCert.pdf
+C0100,CC_calb,0.5867,
+C0100,CC_calc,-1.5627,
+C0100,CC_calt,10.8229,
+C0100,CC_ea434,19706,constant
+C0100,CC_ea620,34,constant
+C0100,CC_eb434,3073,constant
+C0100,CC_eb620,44327,constant
+C0100,CC_sami_bits,12,
+C0100,CC_cal_range,"[204,988]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0100__20231121.csv:11]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0068__20231102.csv (hunk 1 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0068__20231102.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0099__20231120.csv (hunk 2 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0099__20231120.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0100__20231120.csv (hunk 3 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0100__20231120.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0155__20231102.csv (hunk 4 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0155__20231102.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0156__20231107.csv (hunk 5 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0156__20231107.csv:9]: looks good

File: calibration/PHSEND/CGINS-PHSEND-P0158__20231102.csv (hunk 6 above)
Comment (cdobs) [inline calibration/PHSEND/CGINS-PHSEND-P0158__20231102.csv:9]: looks good

**Review by cdobs — APPROVED** [review]
looks good!

**Pull request merged**